DataLink.tex

```diff
@@ -369,31 +325,30 @@ \subsubsection{RESPONSEFORMAT}
 the DALI specification if they chose any formats described there.
 
 
-\subsection{Availability: VOSI-availability}
+\subsection{Registering \blinks~endpoints}
 
-A DataLink web service must have a VOSI-availability endpoint,
-as defined by DALI and VOSI.
+Since normal datalink operations do not involve the Registry, this
+specification poses no requirements to register \blinks~endpoints.
+Datalink clients also generally have no reason to inspect VOSI
+capabilities endpoints, and hence there are no requirements on
+mentioning \blinks~endpoints in any VOSI capability documents.
 
-
-\subsection{Capabilities: VOSI-capabilities}
-
-A standalone DataLink web service must have a
-VOSI-capabilities endpoint as defined by DALI and VOSI.
-The standardID for the \blinks\ endpoint is
+Operators still wishing to declare \blinks~endpoints can do this by
```

Comment:
Do  we want to mention including a links capability in a data collection resource -- to make publisher_did values resolvable? We have prototyped that and made it work internally (I have not yet modified our published registry resource types/content for this)... we use it to find several endpoints that accept publisher_did value as the input, one of which is {links}.